test/byContract/IexecPocoBoost/IexecPocoBoostDelegate.test.ts

```diff
@@ -2836,46 +2598,69 @@ describe('IexecPocoBoostDelegate', function () {
             anyone,
         ).viewDealBoost(dealId);
     }
-});
 
-/**
- * Create a fake ERC734 identity contract instance.
- * @returns A fake ERC734 identity contract instance.
- */
-async function createFakeErc734IdentityInstance() {
-    return await smock.fake<IERC734>(IERC734__factory);
-}
+    /**
```

Comment:
Note for the future:
I think we can declare variables such as `proxyAddress, iexecPocoBoostInstance, iexecMaintenanceAsAdmin, ...` outside of the `describe` block to put utility functions outside and keep only init and test functions inside `describe`.

Reply:
See [Declare variables before top describe](https://github.com/iExecBlockchainComputing/PoCo/pull/148/commits/6797637efeb6a6444da598f9925e12a83dfe75c4).
Yes, not sure which form we generally prefer, but at least this form reduces diff a bit in [this context](https://github.com/iExecBlockchainComputing/PoCo/pull/148/files)  :+1: !